src/components/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
We should talk about folder structure. Like snapshots, I prefer having a folder called __stories__, but in that case, we have to refactor tests to put them in their specific folders (e.g, __tests__)

Reply:
Do you mean something like? 

```
src/
│ 
│─── components
   │─── Icon
      │─── stories
      │─── tests
         │─── __snapshots__
```

Reply:
Something like that, but with a unification of both folders (tests and stories), prefixing and suffixing using double underscores. What do you think about?
Also, I would like to hear @priscilawebdev and @juanpicado (but this talk could be extracted to an issue and discuss there :wink: )

Reply:
thank you for asking @sergiohgz ...Hmm 🧐, I liked the @FilipMessa approach. I like to always keep tests together with it's component/container .. what do you think @juanpicado?

Reply:
Agree with @priscilawebdev

Reply:
For more context, usuall the `__xxx__` are intended for tests or fixtures. Stories are non both of them, but, let's experiment for now, I think should be fine. If we realize is not a good approach, we can change it in the future.

Reply:
Let's keep as it's now for simplicity. We can discuss later if folders contains too much things :wink: